locales/en/common.json

```diff
@@ -25,5 +26,14 @@
   "type_value_to_confirm_plain": "Type {{value}} to confirm",
   "unexpected_error": "There was a problem processing the request. Please try again.",
   "updated": "Updated",
-  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for."
+  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for.",
+  "bootstrap_server": "Bootstrap server",
+  "page": "Page",
+  "sasl_plain": "SASL/PLAIN",
+  "token_endpoint_url": "Token endpoint URL",
+  "more_info_about_sasl_oauthbearer": "More information about SASL/OAUTHBEARER",
+  "sasl_oauthbearer": "SASL/OAUTHBEARER",
+  "recommended": "Recommended",
+  "authentication_method": "Authentication method"
```

Comment:
The strings you're adding here are contents that display on the Connection tab. Do we have guidelines that describe what makes a string a `common` string? Something seems odd about separating strings like
`bootstrap_server` and `bootstrap_server_description`, or `sasl_plain` and `sasl_plain_description`. 

Should these be moved to "connection_tab": { ... } in kafka.json instead?